py/variable.cpp

```diff
@@ -20,18 +20,32 @@ static PyObject*
 Variable_new( PyTypeObject* type, PyObject* args, PyObject* kwargs )
 {
 	static const char *kwlist[] = { "name", "context", 0 };
-	PyObject* name;
 	PyObject* context = 0;
+
+#if PY_MAJOR_VERSION >= 3
+	const char* name;
```

Comment:
It appears to be owned by Python: https://docs.python.org/3.4/c-api/arg.html#strings-and-buffers